### ISSUE TITLE
[OWL-VIT] Changing the batch size

### ISSUE THREAD

**Issue by assafbot** [google-research/scenic#722]
I'm trying to train the model but getting out-of-memory error due to batch size.

1. How many GPUs did you use for training?
2. Did you use a single machine? if not, how can I execute the training on multiple machines?
3. If I have to change the batch size, do I need to change the LR or any other parameter accordingly?

**Comment by mjlm** [comment]
We usually trained on 64 TPUv2 or v3 chips (8 machines/hosts with 8 chips each).

In general, if you decrease the batch size, you may have to decrease the learning rate, but I'd tune that experimentally. We found that larger batch sizes typically lead to better results.

The [training code](https://github.com/google-research/scenic/blob/main/scenic/projects/owl_vit/trainer.py) is compatible with multi-host training since it uses `jax.pmap`. Multi-host training is a fairly complex topic and we don't have the bandwidth to provide support for that here, but you can find some information at https://jax.readthedocs.io/en/latest/multi_process.html.

**Comment by aliencaocao** [comment]
What are the modification needed to the training script of OWL-VIT to work on a multi-gpu single machine setup?
